src/induction/base.ml

```diff
@@ -326,6 +396,11 @@ let init input_sys aparam trans =
     (Some (SMTSolver.declare_fun solver)) trans Numeral.zero
   |> SMTSolver.assert_term solver ;
 
+  (* Function 'skip_steps_init' has the side effect of skipping steps
+      for reachability queries if we know we have to unroll the transition
+      relation many times before we can find an example trace. *)
+  let num_skip = if no_skip then 0 else skip_steps_init trans solver in
+
```

Comment:
This is not necessary, it should always start with `k=0`. The call to `skip_steps_next` in `next` should take care of skipping steps.